addons/mail/static/src/models/call_main_view.js

```diff
@@ -123,6 +123,23 @@ registerModel({
         _computeIsControllerFloating() {
             return Boolean(this.callView.isFullScreen || this.callView.activeRtcSession && !this.callView.threadView.compact);
         },
+        /**
+         * @private
+         * @returns {FieldCommand}
+         */
+        _computeTiles() {
+            if (this.callView.activeRtcSession) {
```

Comment:
Name of `activeRtcSession` not clear enough I think (maybe rename in another PR)